### PR TITLE
Merge feauture-signup to branch develop

### PR DESCRIPTION
**Pull request by ampaire**
This Pull request presents html forms for the intuit sign up page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <body>
+        <header>
+            <div class="nav-text">
+                <p>Already have an Intuit account?</p>
+                <a href="#">I forgot my User ID or Password</a>
+            </div>
+            <button class="nav-btn">Sign in</button>
+        </header>
+        <div class="image-logo">
+            <img src="images/intuitLogo.png" alt="Intuit Logo">
```
Comment (samgaco) [inline index.html]: This image should not contain the texts with the logos below.

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @ampaire and @Elijahscriptdev :hand: 

### Everything looking nice so far :clap:  some things to work on before we can give you the approval:

* This part here should be composed by text (mint, quickbooks and turbotax) and the logos aligned at the left.
Screenshot_2019-08-16_23-05-42
* You could add a **live demo** of the site to your readme file, using https://raw.githack.com/ for example.
* Read on [HTML semantic ](https://www.w3schools.com/html/html5_semantic_elements.asp )elements and implement them in your project. For example, you should use the `<main>` tag for the dominant content in the body of your project.
* Read about [git flow ](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) and apply it for your next projects. (This pull request should be merging to the master branch)

> You will need to create a Pull Request to merge your feature branch with the master branch and submit the link to that Pull Request using a form linked in the lessons’ description. This way, you will get a code review from one of our Technical Support Engineers 

### That's it. Please :pray: submit back for another code review when you are done with these changes :muscle:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        </div>
+        <p class="form-bottom">Invisable reCAPTCHA by Google <a href="#">Privacy Policy</a> and <a href="#">Terms of Use</a>.</p>
+        <footer>
+            <a href="#">Legal</a>
+            <a href="#">Privacy</a>
+            <a href="#">Security</a>
+            <p>&copy; 2019 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.</p>
+            <p>Terms and conditions, features, support, pricing, and service options subject to change without notice.</p>
+        </footer>
+        </main>
```
Comment (stratospherique) [inline index.html:62]: the footer should be sibling to main and not a children of it.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+.logo img{
+    width: 100px;
+    margin-left: 600px;
+}
+.image-logo{
+    width: 400px;
+    margin-left: 500px;
+    margin-top: 30px;
+    margin-bottom: -130px;
+}
```
Comment (stratospherique) [inline styles.css:45]: The logos with their text are not aligned with the proper spacing. They should look like this:.
image

**Review by stratospherique — CHANGES_REQUESTED** [review]
Good effort :clap: and almost done . However, you should fix the following issues:

File: styles.css (hunk 3 above)
Comment (stratospherique) [reply styles.css:45]: And that's It. When finished! kindly re-submit your Pull request for another code review. :muscle: :muscle:

**Pull request merged**